CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to [bpmn-js](https://github.com/bpmn-io/bpmn-js) are documen
 
 ___Note:__ Yet to be released changes appear here._
 
+## 17.9.3
```

Comment:
Let's not add the next version yet and keep this in the unreleased changes section. We don't know what exactly what the next version will be. It also might be a minor or major.